test/number.js

```diff
@@ -262,8 +266,10 @@ describe('Number types', function() {
 
       v
         .isValid(0)
-        .should.eventually()
-        .equal(true),
+        .should.be.rejected()
+        .then(null, function(err) {
+          err.errors[0].should.contain('this must be a positive number');
```

Comment:
this isn't hit i don't think. `should.be.rejected` will catch the error and return it instead of letting it propagate a believe

Reply:
yup my bad, I did it without testing locally. Commit coming down the pipeline...